.github/workflows/vcs.yml

```diff
@@ -26,7 +26,11 @@ jobs:
 
     - name: Get latest release tag name
       run: |
-        API_RELEASE_URL=$(echo "${{ github.event.pull_request.base.repo.releases_url }}")
+        if [ "pull_request" == "${{ github.event_name }}" ]; then
+            API_RELEASE_URL=$(echo "${{ github.event.pull_request.base.repo.releases_url }}")
+        else
+            API_RELEASE_URL=$(echo "${{ github.event.repository.releases_url }}")
```

Comment:
I don't know well about it has different value when main branch and PR status.
But if they're have same value how about remove if condition and just use `github.event.repository.release_url`?

Reply:
The config is different for type pull_request and push. It was discovered with debug step https://github.com/babenek/CredSweeper/commit/8848aa46268b3092554c8ab6740ac3e21c210c67